siliconcompiler/tools/openroad/rcx_extract.py

```
@@ -10,6 +10,18 @@ def __init__(self):
 
         self.add_parameter("corner", "str", "Parasitic corner to generate RCX file for")
 
+    def set_openroad_rcxcorner(self, corner: str,
+                               step: str = None, index: str = None):
```

Ruff (0.14.10)

PEP 484 prohibits implicit `Optional`

Convert to `Optional[T]`

(RUF013)